### DISCUSSION TITLE
지도부분 Scrap 관련 예외 상황 발생

### DISCUSSION THREAD

**Discussion by Haroldfromk** [SwiftBreakers/TTEOPPOKKI4U#60] [General]
![CleanShot 2024-06-18 at 05 21 59@2x](https://github.com/SwiftBreakers/TTEOPPOKKI4U/assets/97341336/4647b2f2-52a5-487f-912e-579dfb9af6a8)

현재 지도에서 flag를 누를경우 title의 label이 "" 로 받아서 제대로 scrap이 안되는것 같아요. 확인 부탁드립니다 :)

**Comment by labydin** [comment]
이전 방식으로 돌려서 해결했습니다! merge해 두겠습니다. 알려주셔서 감사해요 :)
